src/hearings/containers/request-hearing/location-search-container/location-search-container.component.ts

```diff
@@ -1,7 +1,8 @@
-import { Component, OnInit } from '@angular/core';
-import { FormBuilder, FormGroup } from '@angular/forms';
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, RequiredValidator, Validators } from '@angular/forms';
+import { SearchLocationComponent } from '@hmcts/rpx-xui-common-lib';
 import { Store, select } from '@ngrx/store';
-import { LocationByEPIMSModel } from '@hmcts/rpx-xui-common-lib/lib/models/location.model';
+import { LocationByEPIMSModel } from 'api/locations/models/location.model';
```

Comment:
```suggestion
import { LocationByEPIMSModel } from '@hmcts/rpx-xui-common-lib/lib/models/location.model';
```
Don't import api models